### PR TITLE
sync string and number fields

### PR DESCRIPTION
**Pull request by jjidt**
also adds number length limit

### CODE REVIEW

File: sync/src/app.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ app.use(logger('dev'))
 app.use(json())
 app.use(urlencoded({ extended: true }))
 app.use(cookieParser())
-
```
Comment (dheniges) [inline sync/src/app.js:13]: It's a good change, it's a good change.
https://www.youtube.com/watch?v=Rt3gcSyL5vg

**Review by dheniges — CHANGES_REQUESTED** [review]
It works in the update scenario, but some issues I came across. The 2nd and 3rd issues may be separate from this PR, in which case they can go in Asana.

- Two react warnings, see screenshots below
- When user 1 adds a new string or number, user 2 doesn't see it until user 1 enters a value
- Reordering fields isn't reflected for other users until refresh

Screen Shot 2020-03-04 at 8 20 06 AM
Screen Shot 2020-03-04 at 8 20 31 AM

**Comment by jjidt** [comment]
It made sense to go ahead and fix sync for create/remove/reorder, check it :shockandawe:

**Review by dheniges — APPROVED** [review]
What can I say? It syncs :tada: :realjj:

**Pull request merged**